modules/ossm-smcp-prod.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deploy-production.adoc
+
+
+[id="ossm-smcp-prod_{context}"]
+= Configuring your `ServiceMeshControlPlane` resource for production
+
+If you have installed a basic `ServiceMeshControlPlane` resource to test {ProductShortName}, you must configure it to production specifications before your deploy your production service mesh.
+
+You cannot change the `metadata.name` field of an existing `ServiceMeshControlPlane` resource. 
+ 
+For production deployments you _must_ customize the default template. Follow this procedure to find and save your staging `ServiceMeshControlPlane` resource by using the CLI.
+
+Follow this procedure to create or edit the `ServiceMeshControlPlane` with the command line.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. Enter the following command. Then, enter your username and password when prompted.
++
+[source,terminal]
+----
+$ oc login https://{HOSTNAME}:6443
+----
++
+. Change to the project where you installed the control plane, for example istio-system.
++
+[source,terminal]
+----
+$ oc project istio-system
+----
++
+. Retrieve the current `ServiceMeshControlPlane` resource and save it locally to edit it.
++
+[source,terminal]
+----
+oc get smcp <smcp_name> -o yaml > <smcp_name>.v2.yaml
+----
++
+. Edit the `ServiceMeshControlPlane` file. Change the `metadata.name` field to `production`.
++
+. Configure Jaeger for production. To use the `production` deployment strategy, set  `spec.addons.jaeger.install.storage.type` to 'Elasticsearch' and specify additional configuration options under `install`. Or, you can create and configure your Jaeger instance and set `spec.addons.jaeger.name` to the name of the Jaeger instance, for example `jaeger-production`.
++
+.Control Plane default Jaeger parameters (Elasticsearch)
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: production
+spec:
+  version: v2.0
+  tracing:
+    sampling: 10000
```

Comment:
I don't think we want 100% tracing for a production scenario.  The default is probably okay, which I think is 1%.